src/twisted/web/test/test_http_headers.py

```diff
@@ -371,6 +371,9 @@ def test_getRawHeadersDefaultValue(self):
         h = Headers()
         default = object()
         self.assertIdentical(h.getRawHeaders(u"test", default), default)
+        self.assertIdentical(h.getRawHeaders(u"test", None), None)
+        self.assertEqual(h.getRawHeaders(u"test", [None]), [None])
+        self.assertEqual(h.getRawHeaders(u"test", [u"\N{SNOWMAN}"]), [u"\N{SNOWMAN}"])
```

Comment:
I think Twisted should have testtools-style `expectThat` APIs or should use Hypothesis to make it easier to make a lot of assertions without an explosive number of test methods. :)

I left these assertions in this existing test method because the prevailing convention in this module is to make multiple similar assertions in a single test method.

If there's a project policy to adhere to the overall convention rather than reflect local convention, I can update this new code to do so.